packages/material-ui/src/Stepper/Stepper.js

```diff
@@ -86,7 +94,7 @@ function Stepper(props) {
           index,
           ...state,
         }),
-      React.cloneElement(step, { ...controlProps, ...step.props, ...state }),
+      React.cloneElement(step, { ...controlProps, ...step.props, ...state, prevStepState }),
```

Comment:
Could you add this to `controlProps`. It makes it easier to follow where each prop is coming from and what the priority is.

Reply:
done